README.md

```diff
@@ -29,15 +29,15 @@ edX and Tutor version you are deploying. If you are installing this
 plugin from a branch in this Git repository, you must select the
 appropriate one:
 
-| Open edX release  | Tutor version      | Plugin branch | Plugin release |
-|-------------------|--------------------|---------------|----------------|
-| Lilac             | `>=12.0, <13`      | Not supported | Not supported  |
-| Maple             | `>=13.2, <14`[^v1] | `maple`       | 0.3.x          |
-| Nutmeg            | `>=14.0, <15`      | `nutmeg`      | 1.x.x          |
-| Olive             | `>=15.0, <16`      | `main`        | 2.x.x          |
-| Palm              | `>=16.0, <17`      | `main`        | 3.x.x          |
-| Quince            | `>=17.0, <18`      | `main`        | 3.x.x          |
-| Redwood[^journal] | `>=18.0, <19`      | `main`        | 3.x.x          |
+| Open edX release  | Tutor version      | Plugin branch | Plugin release    |
+|-------------------|--------------------|---------------|-------------------|
+| Lilac             | `>=12.0, <13`      | Not supported | Not supported     |
+| Maple             | `>=13.2, <14`[^v1] | `maple`       | 0.3.x             |
+| Nutmeg            | `>=14.0, <15`      | `nutmeg`      | 1.x.x             |
+| Olive             | `>=15.0, <16`      | `olive`       | 2.x.x             |
+| Palm              | `>=16.0, <17`      | `quince`      | `>=2.1.0, <3.4.0` |
+| Quince            | `>=17.0, <18`      | `quince`      | `>=3.1.0, <3.4.0` |
+| Redwood[^journal] | `>=18.0, <19`      | `main`        | `>=3.3.0`         |
```

Comment:
I *think* that since we are effectively dropping support for any release prior to Redwood, we should bump the major number on the next release. Which would mean that it should be this instead:

```suggestion
| Palm              | `>=16.0, <17`      | `quince`      | `>=2.1.0, <4` |
| Quince            | `>=17.0, <18`      | `quince`      | `>=3.1.0, <4` |
| Redwood[^journal] | `>=18.0, <19`      | `main`        | `>=4`         |
```

... or can you think of a reason why that would be a terrible idea?

Reply:
Well a major release bump makes sense yes, but for Redwood we should still set `>=3.3.0` to match the Changelog entries, no?

Reply:
I don't see that necessity, to be honest. That we *started* to support Redwood in 3.3.0 doesn't keep us from saying we *now* only want to support it with 4.0.0 forward. Wouldn't you agree?

Reply:
Ok, sure.